+mystica/+viz/@VisualizerMatlab/rotateCameraView.m

```diff
@@ -30,3 +30,15 @@ function rotateCameraView(obj,durationTotal,durationInitialPhase,durationFinalPh
         finalPos = angleIncrement;
     end
 end
+
+function q = wrapDegTo180(q)
+    if abs(q)>=360
+        q = mod(q,360);
+    end
+    if q > 180
+        q = q - 360;
+    end
+    if q < -180
```

Comment:
Don't you need an equal in either one of the two comparisons? Otherwise you can get both -180.0 and 180.0, even if they refer to the same angle.

Reply:
Yes, you are right. It's better to fix this

Reply:
Fixed in https://github.com/ami-iit/mystica/commit/11ba649d7b34a80973cde6ca3d57b2b2dceec2dc